models/contracts/chains/contract_creator_project_mapping/contracts_ethereum_contract_creator_project_mapping.sql

```diff
@@ -0,0 +1,16 @@
+ {{
+  config(
+        tags = ['dunesql'],
+        alias = alias('contract_ethereum_creator_project_mapping'),
```

Comment:
file name should mimic schema / alias. with the current file name:
- please add `schema` property above alias, make it `contracts_ethereum`
- update `alias` to `contract_creator_project_mapping`

this will apply univerally

Reply:
we can likely also remove a layer in the directory path, to just be:
`models/contracts/chains/`

since that extra directory layer is just the alias / part of file name